### ISSUE TITLE
Could not resolve dependencies for project io.phasetwo.keycloak:keycloak-magic-link:jar:0.1-SNAPSHOT: Could not find artifact org.keycloak:keycloak-testsuite-utils:jar:17.0.1 in central (https://repo.maven.apache.org/maven2)

### ISSUE THREAD

**Issue by feraudet** [p2-inc/keycloak-magic-link#1]
Hello,

I get this error when trying to compile this project : 
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.758 s
[INFO] Finished at: 2022-05-12T21:42:20+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project keycloak-magic-link: Could not resolve dependencies for project io.phasetwo.keycloak:keycloak-magic-link:jar:0.1-SNAPSHOT: Could not find artifact org.keycloak:keycloak-testsuite-utils:jar:17.0.1 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Any idea ?

Thanks in advance

**Comment by D3f3nd** [comment]
Hello ,

I got the same error.
Build on W10 with:
- Maven 3.8.5
- JAVA 1.8.0_321

Thanks

**Comment by xgp** [comment]
Resolved in [1f6e251](https://github.com/p2-inc/keycloak-magic-link/commit/1f6e251013a5c3ea2c520e65cf948192ce0d782a)
